docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,544 @@
+
+```yaml
+tags:
+  - name: Invoice Items
+    description: |+
+      Giant long explanation about what this business concept is, because other people _might_ not have a clue!
+```
+
+## oas2
+
+These OpenAPI v2.0-only rules can be loaded with `extends: "spectral:oas2"` and include all the oas rules.
+
+
+### operation-formData-consume-check
+
+Operations with an `in: formData` parameter must include `application/x-www-form-urlencoded` or `multipart/form-data` in their `consumes` property.
+
+**Recommended:**  Yes
+
+### api-host
+
+OpenAPI `host` must be present and non-empty string.
+
+**Recommended:** Yes
+
+### api-schemes
+
+OpenAPI host `schemes` must be present and non-empty array.
+
+**Recommended:** Yes
+
+### host-not-example
+
+Server URL should not point at `example.com`.
+
+**Recommended:** Yes
+
+### host-trailing-slash
+
+Server URL should not have a trailing slash.
+
+**Recommended:** Yes
+
+### model-description
+
+Definition `description` must be present and non-empty string.
+
+**Recommended:** No
+
+### operation-security-defined
+
+Operation `security` values must match a scheme defined in the `securityDefinitions` object.
+
+**Recommended:** Yes
+
+### valid-example
+
+Examples must be valid against their defined schema.
+
+**Recommended:** Yes
+
+### oas2-anyOf
+
+OpenAPI v3 keyword `anyOf` detected in OpenAPI v2 document.
+
+**Recommended:** Yes
+
+### oas2-oneOf
+
+OpenAPI v3 keyword `oneOf` detected in OpenAPI v2 document.
+
+**Recommended:** Yes
+
+### oas2-schema
+
+Validate structure of OpenAPI v2 specification.
+
+**Recommended:** Yes
+
+## oas3
+
+These OpenAPI v3.0-only rules can be loaded with `extends: "spectral: oas3"` and include all the oas rules.
+
+### api-servers
+
+OpenAPI `servers` must be present and non-empty array.
+
+**Recommended:** Yes
+
+Share links to any and all servers that people might care about. If this is going to be given to internal people then usually that is localhost (so they know the right port number), staging, and production.
+
+``` yaml
+servers:
+  - url: https://example.com/api
+    description: Production server
+  - url: https://staging.example.com/api
+    description: Staging server
+  - url: http://localhost:3001
+    description: Development server
+```
+
+If this is going out to the world, maybe have production and a general sandbox people can play with.
+
+
+### model-description
+
+Model `description` must be present and non-empty string.
+
+**Recommended:** No
+
+### operation-security-defined
+
+Operation `security` values must match a scheme defined in the `components.securitySchemes` object.
+
+**Recommended:** Yes
+
+### server-not-example.com
+
+Server URL should not point at `example.com`.
+
+**Recommended:** No
```

Comment:
How come oas2 `host-not-example` is tagged as Recommended and this one isn't?

Reply:
Docs mistake! checked the rest of oas2 and its good.